nebula/ui/components/VPNServerList.qml

```diff
@@ -237,6 +240,19 @@ FocusScope {
                         }
                     }
                 }
+
+                Timer {
+                    id: recommendedLocationsRefreshTimer
+                    interval: 2000
+                    running: VPNServerLatency.isActive
+                    repeat: false
```

Comment:
`repeat` is set to false by default - no need to re-bind it here

Reply:
Done!